tests/clv/test_utils.py

```diff
@@ -137,8 +150,7 @@ def test_customer_lifetime_value_with_known_values(test_summary_data, fitted_bg)
         time=1,
         discount_rate=0.0,
     ).mean(("chain", "draw"))
-    np.testing.assert_almost_equal(clv_d0, expected, decimal=5)
-
+    np.testing.assert_almost_equal(clv_d0, expected, decimal=3)
```

Comment:
I am going to have to take a look at this one. I couldn't see why the results should have changed on a quick look just now.

Reply:
any ideas what might have changed it? I went quickly through the changes and I can't see any seed changes that could affect that

Reply:
The easiest is to just print all the intermediate quantities before and after the refactoring, and see what changed where

Reply:
I found it, the test was failing with the precision because I have accidentally overwritten the Dirac's delta c param with the c from alpha.

Reply:
but it was within the test itself, while creating the model_config for it

Reply:
That's great!